server/src/disposition/defaultValue.disposition.pipe.ts

```diff
@@ -0,0 +1,14 @@
+import { PipeTransform, Injectable, ArgumentMetadata } from '@nestjs/common';
+
+@Injectable()
+export class defaultValueDispositionPipe implements PipeTransform<any, any> {
```

Comment:
The thing about pipes is that they're sorta generic, sorta not. I don't have a problem with this but because these pipes can be decorated onto anything, regardless of whether it's a disposition class, makes the naming here confusing.

Still, I like this approach, but I would keep that in mind.

Reply:
Yeah, good point. I was also thinking of extracting out the default values into a "default values" file, and parameterizing the pipe. What do you think?

Reply:
the pipe would be placed into a new folder like `/pipe`. Like
`server/src/pipe/defaultValue` 

And then the default value file would be specific to disposition:
`server/src/disposition/default-value.ts`

Reply:
That would be cool. I didn't know you could apply decorators in taht way.

My concern is that it's method-specific

Reply:
For example: why is the defaultValue not applied on the get requests?

Reply:
Good Q. We definitely could... In this case the property "dcpNameofpersoncompletingthisfield" is never displayed or edited by the user, so it doesn't matter as much. But otherwise I think there's a bit of design thought involved, and just agreeing as a team to transform certain backend values before they reach the frontend